net/pfSense-pkg-frr/files/usr/local/pkg/frr/frr_bfd_peers.xml

```diff
@@ -0,0 +1,231 @@
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+	]]>
+	</copyright>
+	<name>frr_bfd_peers</name>
+	<title>Services/FRR/BFD</title>
+	<include_file>/usr/local/pkg/frr.inc</include_file>
+	<aftersaveredirect>pkg.php?xml=frr/frr_bfd_peers.xml</aftersaveredirect>
+	<tabs>	
+		<tab>
+			<text>BFD</text>
+			<url>pkg_edit.php?xml=frr/frr_bfd.xml</url>
+		</tab>
+		<tab>
+			<text>Peers</text>
+			<url>pkg.php?xml=frr/frr_bfd_peers.xml</url>
+			<active/>
+		</tab>	
+		<tab>
+			<text>[Global Settings]</text>
+			<url>pkg_edit.php?xml=frr.xml</url>
+		</tab>
+		<tab>
+			<text>[BGP]</text>
+			<url>pkg_edit.php?xml=frr/frr_bgp.xml</url>
+		</tab>
+		<tab>
+			<text>[OSPF]</text>
+			<url>pkg_edit.php?xml=frr/frr_ospf.xml</url>
+		</tab>
+		<tab>
+			<text>[OSPF6]</text>
+			<url>pkg_edit.php?xml=frr/frr_ospf6.xml</url>
+		</tab>
+		<tab>
+			<text>Status</text>
+			<url>/status_frr.php?protocol=bfd</url>
+		</tab>
+	</tabs>
+	<adddeleteeditpagefields>
+		<columnitem>
+			<fielddescr>Peer Address</fielddescr>
+			<fieldname>peer</fieldname>
+		</columnitem>
+		<columnitem>
+			<fielddescr>Description</fielddescr>
+			<fieldname>descr</fieldname>
+		</columnitem>
+		<columnitem>
+			<fielddescr>Label</fielddescr>
+			<fieldname>label</fieldname>
+		</columnitem>
+		<columnitem>
+			<fielddescr>Shutdown</fielddescr>
+			<fieldname>shutdown</fieldname>
+		</columnitem>
+	</adddeleteeditpagefields>
+	<fields>		
+		<field>
+			<name>Peer Configurations</name>
+			<type>listtopic</type>
+		</field>
+		<field>
+			<fielddescr>Peer Address</fielddescr>
+			<fieldname>peer</fieldname>
+			<description>
+				<![CDATA[IP address of a peer]]>
+			</description>
+			<type>input</type>
+			<size>20</size>
+			<required/>
+		</field>
+		<field>
+			<fielddescr>Description</fielddescr>
+			<fieldname>descr</fieldname>
+			<size>50</size>
+			<type>input</type>
+		</field>
+		<field>
+			<fielddescr>Label</fielddescr>
+			<fieldname>label</fieldname>
+			<description>
+				<![CDATA[
+				Label a peer with the provided word. This word can be referenced later on other daemons to refer to a specific peer.
+				]]>
+			</description>
+			<size>50</size>
+			<type>input</type>
+		</field>
+		<field>
+			<fielddescr>Multihop</fielddescr>
+			<fieldname>multihop</fieldname>
+			<description>
+				<![CDATA[
+				 Tell the BFD daemon that we should expect packets with TTL less than 254 (because it will take more than one hop) and to listen on the multihop port (4784).
```

Comment:
Reword so it does not use "we".

"Tell the BFD daemon that we should expect" could be shortened to "Expect" with no loss of meaning.

Also try to reword to eliminate the parenthetical phrase.